internal/operands/tekton-tasks/reconcile.go

```diff
@@ -21,12 +21,12 @@ import (
 // +kubebuilder:rbac:groups=subresources.kubevirt.io,resources=virtualmachines/restart;virtualmachines/start;virtualmachines/stop,verbs=update
 // +kubebuilder:rbac:groups=template.openshift.io,resources=templates,verbs=get;list;watch;create;patch;update;delete
 // +kubebuilder:rbac:groups=template.openshift.io,resources=processedtemplates,verbs=create
-// +kubebuilder:rbac:groups=cdi.kubevirt.io,resources=datavolumes,verbs=*
+// +kubebuilder:rbac:groups=cdi.kubevirt.io,resources=datavolumes,verbs=get;create;delete
 // +kubebuilder:rbac:groups=cdi.kubevirt.io,resources=datasources,verbs=get;create;delete
-// +kubebuilder:rbac:groups=kubevirt.io,resources=virtualmachines/finalizers,verbs=*
-// +kubebuilder:rbac:groups=*,resources=persistentvolumeclaims,verbs=*
-// +kubebuilder:rbac:groups=*,resources=pods,verbs=create
-// +kubebuilder:rbac:groups=*,resources=secrets,verbs=*
```

Comment:
Hi, I already did the same change and changed `groups` to `core` in https://github.com/kubevirt/ssp-operator/pull/641, but your PR also changes verbs which is good. Thanks :)

Reply:
Yeah, I did it following your advice. So I'm the one should thank you :blush: In a follow-up, I plan to review all `groups` and switch them to `core`.